src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -11,135 +11,145 @@ namespace System.Text.RegularExpressions
-        protected internal abstract RegexRunner CreateInstance();
-    }
-    public partial class Regex
+    public partial class Regex : System.Runtime.Serialization.ISerializable
     {
-        public static readonly System.TimeSpan InfiniteMatchTimeout;
-        protected internal string pattern;
-        protected internal RegexRunnerFactory factory;
-        protected internal RegexOptions roptions;
-        protected internal TimeSpan internalMatchTimeout;
+        protected internal System.Collections.Hashtable caps;
+        protected internal System.Collections.Hashtable capnames;
         protected internal int capsize;
         protected internal string[] capslist;
-        protected System.Collections.IDictionary Caps { get; set; }
-        protected System.Collections.IDictionary CapNames { get; set; }
+        protected internal System.Text.RegularExpressions.RegexRunnerFactory factory;
+        public static readonly System.TimeSpan InfiniteMatchTimeout;
+        protected internal System.TimeSpan internalMatchTimeout;
+        [System.CLSCompliant(false)]
+        protected System.Collections.IDictionary Caps { get { throw null; } set { } }
+        [System.CLSCompliant(false)]
+        protected System.Collections.IDictionary CapNames { get { throw null; } set { } }
+        protected internal string pattern;
+        protected internal System.Text.RegularExpressions.RegexOptions roptions;
         protected Regex() { }
+        protected Regex(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public Regex(string pattern) { }
         public Regex(string pattern, System.Text.RegularExpressions.RegexOptions options) { }
         public Regex(string pattern, System.Text.RegularExpressions.RegexOptions options, System.TimeSpan matchTimeout) { }
-        public static int CacheSize { get { return default(int); } set { } }
-        public System.TimeSpan MatchTimeout { get { return default(System.TimeSpan); } }
-        public System.Text.RegularExpressions.RegexOptions Options { get { return default(System.Text.RegularExpressions.RegexOptions); } }
-        public bool RightToLeft { get { return default(bool); } }
+        public static int CacheSize { get { throw null; } set { } }
+        public System.TimeSpan MatchTimeout { get { throw null; } }
+        public System.Text.RegularExpressions.RegexOptions Options { get { throw null; } }
+        public bool RightToLeft { get { throw null; } }
+        public static string Escape(string str) { throw null; }
+        public string[] GetGroupNames() { throw null; }
+        public int[] GetGroupNumbers() { throw null; }
+        public string GroupNameFromNumber(int i) { throw null; }
+        public int GroupNumberFromName(string name) { throw null; }
         protected void InitializeReferences() { }
-        protected internal static void ValidateMatchTimeout(TimeSpan matchTimeout) { }
```

Comment:
When making formatting/style changes in the same PR its good to segregate them to their own commit and indicate which one that is so that the reviewer knows they should just look at the other ones. Otherwise it's hard to see the real changes in all the style cleanup.